Services/Setup/Version310.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Wizzy\Search\Services\Setup;
+
+class Version310
+{
+    private $setupUtils;
+
+    public function __construct(SetupUtils $setupUtils)
+    {
+        $this->setupUtils = $setupUtils;
+    }
+
+    private $defaultConfigs = [
+        "wizzy_autocomplete_configuration/autocomplete_advanced_configuration/show_default_suggestions" => 0,
+        "wizzy_autocomplete_configuration/autocomplete_advanced_configuration/show_default_products" => 0,
+        "wizzy_autocomplete_configuration/autocomplete_advanced_configuration/display_recently_searched_terms" => 0,
+        "wizzy_autocomplete_configuration/autocomplete_advanced_configuration/autocomplete_pinned_products" => 0
```

Comment:
Create for commented code as well. So in next update we just need to activate it.